<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StikkedSettings</class>
 <widget class="QWidget" name="StikkedSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="apiUrlLabel">
     <property name="text">
      <string>API URL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="apiUrlLineEdit">
     <property name="text">
      <string>http://your.stikked/api/create</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Author's name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="titleLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="privateLabel">
     <property name="text">
      <string>Private</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="privateCheckBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="langLabel">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="langComboBox"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="expirationLabel">
     <property name="text">
      <string>Expiration</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="expirationComboBox">
     <item>
      <property name="text">
       <string>30 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 hour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 day</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 weeks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Keep forever</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
